projects/ngx-speculoos/src/lib/routing-tester.spec.ts

```diff
@@ -16,8 +15,9 @@ import { speculoosMatchers } from './matchers';
 class PageComponent {
   page!: number;
 
-  constructor(route: ActivatedRoute) {
-    route.queryParamMap.subscribe(map => (this.page = parseInt(map.get('page'))));
+  constructor() {
```

Comment:
page should be a signal, and we should use toSignal instead of subscribing

Reply:
Looks like you updated it in your PR, so I did not changed it